### PR TITLE
Adding e2e-kubevirt

### PR DESCRIPTION
**Pull request by bardielle**

### CODE REVIEW

File: ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh [hunk 1]
```diff
@@ -6,27 +6,29 @@ set -o pipefail
 
 CONFIG="${SHARED_DIR}/install-config.yaml"
 
-
-KUBEVIRT_BASE_DOMAIN="tenant1.origin-on-kubevirt.dev.openshift.com"
-KUBEVIRT_API_VIP=192.168.123.15
-KUBEVIRT_INGRESS_VIP=192.168.123.20
-KUBEVIRT_KUBECONFIG=${HOME}/.kube/config
+KUBEVIRT_BASE_DOMAIN="testci.tenant1.dev.openshift.com"
```
Comment (ravidbro) [inline ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh:9]: But we don't know yet if we can use that DNS name

**Review by nirarg — APPROVED** [review]

File: ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh [hunk 2]
```diff
@@ -6,27 +6,29 @@ set -o pipefail
...
+KUBEVIRT_API_VIP=10.123.124.15
+KUBEVIRT_INGRESS_VIP=10.123.124.20
+KUBEVIRT_CIDR="10.123.124.0/24"
 KUBEVIRT_NAMESPACE=tenantcluster
-KUBEVIRT_NETWORK_NAME=tenantcluster
+KUBEVIRT_NETWORK_NAME=mynet
 KUBEVIRT_TENANT_STORAGE_CLASS_NAME=standard
-KUBEVIRT_VOLUME_ACCESS_MODE=ReadWriteMany
+KUBEVIRT_VOLUME_ACCESS_MODE=ReadWriteOnce
```
Comment (ravidbro) [inline ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh:16]: Why? this way we won't support live migration

File: ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh (hunk 1 above)
Comment (bardielle) [reply ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh:9]: We will change it - this is only for create the workflow
nothing is triggerint it yet

File: ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh (hunk 2 above)
Comment (bardielle) [reply ci-operator/step-registry/ipi/conf/kubevirt/ipi-conf-kubevirt-commands.sh:16]: GCP not supported MANY

**Review by ravidbro — APPROVED** [review]

**Pull request merged**